android/res/layout/place_page_preview_common.xml

```diff
@@ -38,7 +38,7 @@
       android:textAppearance="@style/MwmTextAppearance.Body1.Secondary"
       android:visibility="gone"
       tools:text="Till 9 PM"
-      tools:visibility="visible"/>
+      tools:visibility="gone"/>
```

Comment:
PTAL
> On Mar 30, 2016, at 3:07 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/place_page_preview_common.xml <https://github.com/mapsme/omim/pull/2630#discussion_r57876967>:
> 
> > @@ -38,7 +38,7 @@
> >        android:textAppearance="@style/MwmTextAppearance.Body1.Secondary"
> >        android:visibility="gone"
> >        tools:text="Till 9 PM"
> > -      tools:visibility="visible"/>
> > +      tools:visibility="gone"/>
> Не нужно
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2630/files/5a93eb2289f825b4c13c00b7f2e62b93dcd46a3d..1d686a43593da0e6f5fe8156a3a1116671c0b09e#r57876967>